.seed-config.yaml

```diff
@@ -1,21 +1,51 @@
 project:
-  name: "Dailymile"
+  name: "GoMiler"
   state: "unstable"
   image: ""
   website: ""
-  oneLiner: "Daily milestone generation for Gitlab"
+  oneLiner: "Easy milestone creation for your favorite project tracking website"
   description: |
     Generate one milestone per day\week\month within your Gitlab project.
     Use for your Gitlab based ToDo list or sprints.
-  version: "v0.0.0"
+  version: "v0.1.0"
   mainLicense: "apache2"
   docsLicense: "CCBySa"
 
 vision:
   type: "community"
   overview: |
     Simple daily milestone generation for Gitlab.
+readme:
+  usageExample: |
+  
+    git clone https://github.com/okkur/gomiler.git
 
+    go get -u github.com/peterhellberg/link
+
+    cd gomiler
+
+    go build main.go
+
+    ./main -advance=10 -namespace=gitnamespace  -project=gitproject -token=123456789 -url=some.thing.eu  -interval=weekly
+    
+    or
+
+    go run main.go --advance=10 --namespace=gitnamespace  --project=gitproject --token=123456789 --url=some.thing.eu  --interval=weekly
+    
+    flags:
```

Comment:
Instead of hardcoding the flags here you can simply give an example of the help command:

```
For more information about available flags:
  gomiler --help
```